runtime/src/common/version.rs

```diff
@@ -1,12 +1,19 @@
+use num_traits::Zero;
 /// Protocol and runtime versioning.
```

Comment:
This should be at the top and changed to:
```suggestion
//! Protocol and runtime versioning.
```

Reply:
Fixed!  This comment already existed before, though.